inloop/statistics/responses.py

```diff
@@ -0,0 +1,22 @@
+from django.http import JsonResponse
+
+
+class FailureJsonResponse(JsonResponse):
```

Comment:
As long as these response classes aren't used elsewhere, they can reside in `views.py`. (I know that I've written some bad examples of this on my own in other places in code 😇).

Reply:
As you probably have seen, I also removed the SuccessJsonResponse because it just replicated the JsonResponse which has the response code 200 by default. I am probably not going to include much more code here anytime soon, so I will merge this as you suggested 👍🏻